static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -1435,18 +1444,33 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
                               </StyledAlert>
                             )
                           }
-                          additionalAction={{
-                            label: 'Notify integration\u{2026}',
-                            option: {
-                              label: 'Missing an integration? Click here to refresh',
-                              value: {
-                                enabled: true,
-                                id: 'refresh_configs',
-                                label: 'Refresh Integration List',
-                              },
-                            },
-                            onClick: this.refetchConfigs,
-                          }}
+                          {...(organization.features.includes(
+                            'messaging-integration-onboarding'
+                          )
```

Comment:
i'd recommend pulling the `organization.features.includes('messaging-integration-onboarding')` into a variable like: `hasMessagingIntegration` or `isMessagingIntegrationEnabled` and use that variable instead of checking for the features each time.